### PR TITLE
Add Changelog for 1.10.0

### PR DESCRIPTION
**Pull request by rgrunber**
Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## 1.10.0 (August 31st, 2022)
+ * enhancement - Search more folders to infer source roots for invisible projects. See [JLS#2176](https://github.com/eclipse/eclipse.jdt.ls/pull/2176).
+ * enhancement - Support for Android projects. See [JLS#923](https://github.com/eclipse/eclipse.jdt.ls/issues/923).
```
Comment (testforstephen) [inline CHANGELOG.md:5]: Experimental support for Android projects. Also mention there is a setting `java.jdt.ls.androidSupport.enabled` for it, and it's automatically enabled in VS Code Insiders.  See https://github.com/redhat-developer/vscode-java/pull/2644

**Comment by testforstephen** [comment]
Add https://github.com/redhat-developer/vscode-java/issues/2577

**Comment by tarekahf** [comment]
When this change will be released?

**Comment by rgrunber** [comment]
> When this change will be released?

The expected release date  is often included in the release entry of the CHANGELOG : https://github.com/redhat-developer/vscode-java/pull/2648/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR3 (August 31st) . Somtimes it may slip by a day or two if we're waiting on some particular contribution.

File: CHANGELOG.md (hunk 1 above)
Comment (rgrunber) [reply CHANGELOG.md]: Done.

**Review by testforstephen — APPROVED** [review]

**Pull request merged**